_press/how-is-18F-funded.md

```diff
@@ -7,8 +7,7 @@ layout: bare
 
 * GSA funds 18F through the [Acquisition Services Fund][1] (ASF), which allows for investment in the development and delivery of products and tools that will be used by other agencies on a reimbursable basis.
 
-* The Economy Act allows 18F to be reimbursed for services provided to other agencies. Once an agreement is in place, we bill on a quarterly basis through [IPACS][2]. [We’ve written more about how our billing options work on our blog.][3]
```

Comment:
good catch, we should remember to update this if we ever finish writing that post.